Anlab.Mvc/Extensions/StringExtensions.cs

```diff
@@ -53,5 +54,22 @@ public static bool IsEmailValid(this string value)
             }
             return Regex.IsMatch(value.ToLower(), emailRegex);
         }
+
+        public static string FormatPhone(this string value)
+        {
+            if (string.IsNullOrWhiteSpace(value))
+            {
+                return value;
+            }
+            var phoneNumberUtil = PhoneNumbers.PhoneNumberUtil.GetInstance();
+            var phoneNumber = phoneNumberUtil.ParseAndKeepRawInput(value, "US");
+            var phone = phoneNumberUtil.Format(phoneNumber, PhoneNumberFormat.NATIONAL);

```

Comment:
Let's make sure this works with international numbers, like a german number might be 001-541-754-3010

Reply:
![image](https://user-images.githubusercontent.com/469010/47112038-04fd1180-d20a-11e8-9e61-b93900888311.png)

Reply:
With is valid for region: